src/index.html

```diff
@@ -10,7 +10,67 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body_style">
+    <form
+    action="/"
+    method="get"
+    data-qa="big"
+    class="testB"
+    >
```

Comment:
```suggestion
    <form
       action="/"
       method="get"
       data-qa="big"
       class="testB"
    >
```
Fix code style everywhere.